ansible/templates/var/www/stickypedia/start-gollum.sh.j2

```diff
@@ -0,0 +1,11 @@
+# {{ ansible_managed }}
```

Comment:
This is duplicate, this script won't be executable

Reply:
done